index.js

```diff
@@ -138,3 +138,41 @@ newSightingForm.addEventListener('submit', function (event) {
 /************************** EVENTS PART 2 JS MINI CHALLENGE ******************************/
 
 
+
+const animalsUl = document.querySelector("#animals")
+animalsUl.addEventListener('click', function(event){
+    if(event.target.matches("button.like-button")){
+        const likesPtag = event.target.previousElementSibling
+        const likes = parseInt(likesPtag.textContent) + 1
+        likesPtag.textContent = `${likes} Likes`
+      
+    } 
+    else if (event.target.matches("button.delete-button")){
+        const sighting = event.target.closest('li')
+        sighting.remove()
+    } 
+    else if(event.target.matches("button.toggle-update-form-button")){
+        const sighting = event.target.nextElementSibling
+       
+       if (sighting.style.display === 'none'){
+           sighting.style.display = 'inline-block'
+       }  else { 
+           sighting.style.display = 'none' 
+           }
+    
+    }
+   
+}) 
+
+
+
+animalsUl.addEventListener('submit', function(event){
+    event.preventDefault()
+    const updateForm = document.querySelector("form.update-form")
+    const li =  event.target.closest('li')
+    const descriptionPtag = li.querySelector('p')
+    descriptionPtag.textContent = event.target[0].value
+
+})
```

Comment:
Great work here as well. You are listening for any form submission that occurs within the animals UL element. Then you make sure to prevent the default action of the form. Then you find the p tag and update it's value based on the form data. Don't forget to change the form field value to the new description. Again, we can break it apart into 2 steps.

```
/***** BONUS DELIVERABLE part 1 *****/

animalsUL.addEventListener("submit", function(event){
    event.preventDefault()
    if (event.target.matches('.update-form')){
        handleUpdateFormSubmission(event)
    }
})

/***** BONUS DELIVERABLE part 2 *****/
function handleUpdateFormSubmission(e){
    e.preventDefault()
    const description = e.target.children[0].value
    const animalDescriptionPTag = e.target.parentElement.children[0]
    animalDescriptionPTag.innerText = description
    e.target.value = description
}
```